### PR TITLE
Add difference indicator to barscale and fix down trend value

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard8**

### CODE REVIEW

**Comment by VWSCoronaDashboard10** [comment]
compilation error: 
```
src/components/layout/NationalLayout.tsx(166,25): error TS2740: Type 'InfectedPeopleDeltaNormalized' is missing the following properties from type 'National': last_generated, proto_name, name, code, and 18 more.
```

File: src/components-styled/difference-indicator.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ function renderTileIndicator(value: DifferenceDecimal | DifferenceInteger) {
           <IconDown />
         </Span>
         <Span fontWeight="bold" mr="0.3em">
-          {splitText[0]}
+          {`${Math.abs(difference)} ${splitText[0]}`}
```
Comment (VWSCoronaDashboard10) [inline src/components-styled/difference-indicator.tsx:103]: nit: possible to render this as follows?
```tsx
{Math.abs(difference)} {splitText[0]}

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: src/pages/landelijk/ziekenhuis-opnames.tsx [hunk 2]
```diff
@@ -67,7 +67,11 @@ const IntakeHospital: FCWithLayout<NationalPageProps> = (props) => {
             source: text.bronnen.rivm,
           }}
         >
-          <IntakeHospitalBarScale data={dataIntake} showAxis={true} />
+          <IntakeHospitalBarScale
+            data={data}
+            showAxis={true}
+            showValue={true}
+          />
```
Comment (VWSCoronaDashboard10) [inline src/pages/landelijk/ziekenhuis-opnames.tsx:74]: nit: You prefer this over `<IntakeHospitalBarScale data={data} showAxis showValue />`?

**Review by VWSCoronaDashboard10 — APPROVED** [review]

File: src/components-styled/difference-indicator.tsx (hunk 1 above)
Comment (VWSCoronaDashboard8) [reply src/components-styled/difference-indicator.tsx:103]: I think then react will need to add {' '} for the space no? I prefer to format strings with backticks.

File: src/pages/landelijk/ziekenhuis-opnames.tsx (hunk 2 above)
Comment (VWSCoronaDashboard8) [reply src/pages/landelijk/ziekenhuis-opnames.tsx:74]: No I don't, this is just how it was already used and I didn't think about it.

**Pull request merged**